packages/cozy-konnector-libs/src/libs/linkBankOperations.js

```diff
@@ -21,6 +21,7 @@ const flatten = require('lodash/flatten')
 const sumBy = require('lodash/sumBy')
 const geco = require('geco')
 const moment = require('moment')
+const cozyClient = require('./cozyclient')
```

Comment:
I think we could use the variable stored in `global`, don't we?

Reply:
I don't know, I don't to use globals. I have no better argument